### PR TITLE
Support for Ubuntu 14.04

### PR DESCRIPTION
**Pull request by Jaanki**

### CODE REVIEW

**Comment by Jaanki** [comment]
Created a new PR. Tried to rebase and squash the previous one but it got closed :(. May be because I picked and squashed wrong commits.

File: wcbench.sh [hunk 1]
```diff
@@ -572,12 +598,20 @@ install_opendaylight()
 
     # Install required packages
     echo "Installing OpenDaylight dependencies"
-    if "$VERBOSE" = true; then
-        sudo yum install -y java-1.7.0-openjdk unzip wget
-    else
-        sudo yum install -y java-1.7.0-openjdk unzip wget &> /dev/null
+    if [[ OS == ubuntu ]]; then
+        if "$VERBOSE" = true; then
+            sudo apt-get install -y java-1.7.0-openjdk unzip wget
```
Comment (dfarrell07) [inline wcbench.sh]: The Java package has a different name in Ubuntu.

**Comment by dfarrell07** [comment]
Thanks for the persistence and following up, @Jaanki. Reviewing this now.

File: wcbench.sh [hunk 2]
```diff
@@ -159,12 +177,20 @@ install_cbench()
 
     # Install required packages
     echo "Installing CBench dependencies"
-    if "$VERBOSE" = true; then
-        sudo yum install -y net-snmp-devel libpcap-devel autoconf make automake libtool libconfig-devel git
-    else
-        sudo yum install -y net-snmp-devel libpcap-devel autoconf make automake libtool libconfig-devel git &> /dev/null
+    if [[ OS == ubuntu ]]; then
+        if "$VERBOSE" = true; then
+            sudo apt-get install -y net-snmp-devel libpcap-devel autoconf make automake libtool libconfig-devel git
```
Comment (dfarrell07) [inline wcbench.sh]: Some of these also have different package names in Ubuntu:

```
E: Unable to locate package net-snmp-devel
E: Unable to locate package libpcap-devel
E: Unable to locate package libconfig-devel
```

**Comment by dfarrell07** [comment]
I have everything cleaned up and pushed to an `ubuntu` branch. Comparison [here](https://github.com/dfarrell07/wcbench/compare/ubuntu). Can you test that against an Ubuntu box (maybe add an Ubuntu Vagrant box?) and let me know if it works?

Thanks!

**Comment by Jaanki** [comment]
Yes, this works good in my Ubuntu machine.
New to Vagrant.  I will try to write a Vagrantfile for Ubuntu and if needed submit here for merging with the master branch.

**Comment by dfarrell07** [comment]
> Yes, this works good in my Ubuntu machine.

Cool, taking your word for it and merging.

> New to Vagrant. I will try to write a Vagrantfile for Ubuntu and if needed submit here for merging with the master branch.

Any update on this? It would be super helpful for making your Ubuntu support accessible to users. I'll raise an issue to track it.

**Comment by dfarrell07** [comment]
> Cool, taking your word for it and merging.

Merged! Sorry again for the lag. Your commit is [on master here](https://github.com/dfarrell07/wcbench/commit/ac40a12f87977dbb77456814afa0cc5c3719e9dd), my refactoring is [here](https://github.com/dfarrell07/wcbench/commit/bc43c8587e1325ff457827082ac0f39c8af9df8f).

> I'll raise an issue to track it.

#76 is the relevant issue.

Thanks for the contribution, @Jaanki! I hope you stick around and flesh out WCBench's Ubuntu support. :+1:

**Pull request closed without merging**